client/src/components/CharityDetalis/Kpis/index.js

```diff
@@ -44,58 +44,129 @@ const Kpis = (props) => {
       <div className="kpi--row">
         <div className="kpi--column">
           <div className="kpi--item">
-            <span>3YR Average Expense & Managemant Ratio<p className="descrip-def">ChariZone would like to see this ratio below 25%</p></span>
+            <span>Expense and Management Ratio</span>
             <div className="progressBar">
               <CircularProgressbar
                 strokeWidth="13"
                 percentage={EMR.replace('%', '')}
                 text={EMR} />
             </div>
+            <div className="minMax">
+                <div className="innerMaxMin">
+                  <p>Flag over 20%</p>
+                  </div>
+                </div>
+            <div className="overlayInner">
+            <p className="descrip-def innerDef">Effective charities must continuously develop their strategy, invest in their infrastructure and attract talent. However, these administrative expenses remain reasonable and in line with its charitable mission.</p>
+            </div>
           </div>
           <div className="kpi--item">
-            <span>3Y Average Fundraising Effecincy</span>
+            <span>Fundraising Efficiency</span>
             <div className="progressBar">
               <CircularProgressbar
                 strokeWidth="13"
                 percentage={averageFundraising.replace('%', '')}
                 text={averageFundraising}
               />
+                <div className="minMax">
+                <div className="innerMaxMin">
+                  <p>Flag over 20%</p>
+                </div>
+              </div>
+               <div className="overlayInner">
+            <p className="descrip-def innerDef">Charities must be efficient fundraisers, spending less to raise more. We calculate this ratio by determining how much it spends to generate £1 in voluntary income.</p>
+            </div>
             </div>
           </div>
         </div>
         <div className="kpi--column">
           <div className="kpi--item">
-            <span>Efficiency And Capacity Ratio</span>
+            <span>Capacity Ratio</span>
             <div className="progressBar">
               <CircularProgressbar
                 strokeWidth="13"
                 percentage={Ecr.replace('%', '')}
-                text={Ecr}
-              />
+                text={Ecr} />
+            </div>
+            <div className="minMax">
+                <div className="innerMaxMin">
+                  <p>Min</p>
+                  <p>10.0%</p>
+                </div>
+                <div className="innerMaxMin">
+                <p>Max</p>
+                <p>-10.0%</p>
+                </div>
+              </div>
```

Comment:
@OnsNader I wonder if you could make a small helper function to create this `minMax` block since it's a similar block used in a few different places.  Something like you all did with `returnFlag`, so that you could just write `{minMax('10.0%', '-10.0%')}`.